app/touch-bar-menu.js

```diff
@@ -3,7 +3,7 @@
 const { TouchBar } = require('electron')
 
 const { TouchBarButton, TouchBarLabel, TouchBarSpacer } = TouchBar
-const MAX_TITLE_LENGTH = 38
+const MAX_TITLE_LENGTH = 39
```

Comment:
@salomvary I recreated the icons using the exact assets from Soundcloud. They are actually in SVG which you seemed to want earlier. Perhaps that would help.

I tested the title length and 39 is actually the maximum, the increase buttons seemed to have no effect. It seems although the 39 char length has more to do with the electron implementation (a bug possibly?)